### ISSUE TITLE
Create a Participant Portal as a GUI for the Participant IdP

### ISSUE THREAD

**Issue by sh-ura** [dycons/participant-portal#4]
As a participant, I should be able to authenticate my identity by logging into a Participant Portal. I expect to be prompted for my `username`, `password` credentials by a `Log In` page. Upon submission of credentials, I expect to be prompted to re-enter my credentials if authentication fails, or receive a session JWT if authentication succeeds.

### Acceptance Criteria

- The Participant IdP and Participant Portal can be launched *together* by running `docker-compose up`. If there are any other setup steps required, they are well documented and fully reproducible by the reviewer.
- Once launched, the Participant Portal posts a `Log In` page. The user is prompted to authenticate with the Participant IdP by submitting credentials entered into this page.
- If the user's credentials do not validate against the Participant IdP, the user is redirected back to the `Log In` page with the following message: "This username and password do not match a known user. Please verify and re-enter your credentials."
- If the user's credentials are validated by the Participant IdP, the Participant Portal receives a JWT for the user and stores it locally. The user is then redirected to a page displaying their JWT.

### Authentication Schema

User submits:
- username : varchar
- password : varchar

User receives a `JWT` containing:
- admin_identifier : varchar

### Ticket dependencies
dycons/participant-portal#1 Must be completed first.

**Comment by yuriy-pavlov** [comment]
WON'T DO
Closing due to repo archival.
